src/org/jitsi/impl/configuration/ConfigurationServiceImpl.java

```diff
@@ -102,7 +103,8 @@
      * and attempts to do so will simply be ignored.
      * @see #defaultProperties
      */
-    private Map<String, String> immutableDefaultProperties = new HashMap<>();
+    private Map<String, String> immutableDefaultProperties
+        = new ConcurrentHashMap<>();
```

Comment:
Thanks, Lyubo! I was trying to stay on the safe side and did not always follow all the usages. I'll remove the commits that change ConfigurationServiceImpl, MediaServiceImpl, MediaFormatImpl and ParameterizedVideoFormat.